code/game/machinery/washing_machine.dm

```diff
@@ -234,6 +234,16 @@ GLOBAL_LIST_INIT(dye_registry, list(
 	WM.bloody_mess = TRUE
 	gib()
 
+/obj/item/organ/brain/machine_wash(obj/machinery/washing_machine/WM)
+	var/datum/antagonist/brainwashed/BW = brainmob.mind.has_antag_datum(/datum/antagonist/brainwashed)
+	if(BW)
+		BW.objectives = list()	//washed clean!
+	if(prob(33))
+		gain_trauma_type(/datum/brain_trauma/mild/dumbness, TRAUMA_RESILIENCE_SURGERY)
+		desc = "A piece of juicy meat found in a person's head. This one is quite smooth and smells like detergent."
+	applyOrganDamage(rand(10, 60), BRAIN_DAMAGE_DEATH)	//You are putting a brain into a washing machine, what did you expect?
+	brainwash(brainmob, pick(world.file2list("strings/abductee_objectives.txt")))
```

Comment:
I don't like that people have a way to brainwash themselves. Perhaps it should be limited to objectives that are solely bad for the user and complete non-destructive.

Reply:
I will make it work properly after the exams.